src/igvfd/schemas/file.json

```diff
@@ -154,7 +154,7 @@
             }
         },
         "transcriptome_annotation": {
-            "comment": "The assembly shoud be consistent with annotation versions of respective species. Annotation version should not be submitted for custom assemblies.",
+            "comment": "The assembly shoud be consistent with annotation versions of respective species. Annotation version should not be submitted for custom assemblies. This dependency is not relevant for matrix files.",
```

Comment:
I was debating if this part should be moved to shared_dependencies.json instead, ended up with leaving a comment here for now since we have another ticket investigating all files on these two properties.